### PR TITLE
Add Commento to Easypanel 🎉

### PR DESCRIPTION
**Pull request by Supernova3339**
Shoutouts to ITzDerock for the fix with postgres!
Your welcome everyone :)

### CODE REVIEW

File: templates/commento/index.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        type: "image",
+        image: input.enableCommentoPlusPlus && !input.appServiceImage
+          ? 'ghcr.io/souramoo/commentoplusplus:v1.8.7'
+          : (input.appServiceImage || 'registry.gitlab.com/commento/commento:latest'),
+      },
+      proxy: {
+        port: 8080,
+        secure: true,
+      },
+          domains: [
```
Comment (deiucanta) [inline templates/commento/index.ts:29]: Formatting issues

**Comment by deiucanta** [comment]
This is a nice app. Thanks for the contribution!

Please remove the lock files and fix the formatting. I will merge the PR after that.

**Comment by Supernova3339** [comment]
> This is a nice app. Thanks for the contribution!
> 
> Please remove the lock files and fix the formatting. I will merge the PR after that.

what lock files and which to fix formatting

File: templates/commento/index.ts (hunk 1 above)
Comment (Supernova3339) [reply templates/commento/index.ts:29]: What do you mean

**Comment by Supernova3339** [comment]
> > This is a nice app. Thanks for the contribution!
> > 
> > Please remove the lock files and fix the formatting. I will merge the PR after that.
> 
> what lock files and which to fix formatting

yarn.lock will be removed but not sure how to fix formatting. It works, so I ain't sure what needs fixes

**Comment by Supernova3339** [comment]
fixing the dumb fix

**Pull request closed without merging**